api.js

```diff
@@ -10,6 +11,7 @@ config.cwd = process.__dirname;
 
```

Comment:
Are we no longer changing the API initialization flow? Or are the changes in the startup scripts what you were talking about?